frontend/src/modules/entitydetails/components/Helpers.js

```diff
@@ -47,36 +41,23 @@ export default class Helpers extends React.Component<Props> {
     render() {
         const {
             entity,
-            history,
             isReadOnlyEditor,
-            isTranslator,
             locale,
             machinery,
             otherlocales,
             orderedOtherLocales,
             preferredLocalesCount,
             parameters,
             user,
-            deleteTranslation,
             updateEditorTranslation,
-            updateTranslationStatus,
             searchMachinery,
         } = this.props;
 
-        const historyCount = history.translations.length;
         const machineryCount = machinery.translations.length;
         const otherlocalesCount = otherlocales.translations.length;
 
         return <Tabs>
             <TabList>
-                <Tab>
-                    <Localized id='entitydetails-Helpers--history'>
```

Comment:
This can be removed from our various .ftl files.

Reply:
Good point!